src/main/java/de/bayern/gdi/gui/Controller.java

```diff
@@ -598,8 +598,15 @@ protected void handleSaveConfig(ActionEvent event) {
             }
 
             FileChooser fileChooser = new FileChooser();
+            FileChooser.ExtensionFilter xmlFilter =
+                    new FileChooser.ExtensionFilter("xml files (*.xml)", "xml");
+            fileChooser.getExtensionFilters().add(xmlFilter);
+            fileChooser.setSelectedExtensionFilter(xmlFilter);
             fileChooser.setTitle(I18n.getMsg("gui.save-conf"));
             File configFile = fileChooser.showSaveDialog(getPrimaryStage());
+            if (!configFile.toString().endsWith(".xml")) {
+                configFile = new File(configFile.toString() + ".xml");
+            }
```

Comment:
What if the user decides to change the name "config.xml" to "myspecialconf"? It seems, that I can't force the extension ".xml" in the frontend (on Linux at least), so the user won't get an aknowledgement of the kind "Hey, myspecialconf.xml already exists, do you want to overwrite it? He only would get a message, when there is already a file calles myspecialconf.